js/wazirx.js

```diff
@@ -0,0 +1,709 @@
+        //         "price": "9293.0",
+        //         "origQty": "10.0",
+        //         "executedQty": "8.2",
+        //         "status": "cancel",
+        //         "type": "limit",
+        //         "side": "sell",
+        //         "createdTime": 1499827319559,
+        //         "updatedTime": 1499827319559
+        //     },
+        //     {
+        //         "id": 30,
+        //         "symbol": "wrxinr",
+        //         "price": "9293.0",
+        //         "stopPrice": "9200.0",
+        //         "origQty": "10.0",
+        //         "executedQty": "0.0",
+        //         "status": "cancel",
+        //         "type": "stop_limit",
+        //         "side": "sell",
+        //         "createdTime": 1499827319559,
+        //         "updatedTime": 1507725176595
+        //     }
+        // ]
+        const orders = this.parseOrders (response, market, since, limit);
+        return orders;
+    }
+
+    async cancelAllOrders (symbol = undefined, params = {}) {
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' cancelAllOrders requires a `symbol` argument');
+        }
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'symbol': market['id'],
+        };
+        return await this.spotV1PrivateDeleteOpenOrders (this.extend (request, params));
+    }
+
+    async cancelOrder (id, symbol = undefined, params = {}) {
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' cancelOrder requires a `symbol` argument');
+        }
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'symbol': market['id'],
+            'orderId': id,
+        };
+        const response = await this.spotV1PrivateDeleteOrder (this.extend (request, params));
+        return this.parseOrder (response);
+    }
+
+    async createOrder (symbol, type, side, amount, price = undefined, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'symbol': market['id'],
+            'side': side,
+            'type': type,
+            'quantity': amount,
+        };
+        if ((type === 'limit') || (type === 'stop_limit')) {
+            if (price === undefined) {
+                throw new ArgumentsRequired (this.id + ' createOrder requires a price argument for ' + type + ' orders');
+            }
+            request['price'] = this.priceToPrecision (symbol, price);
+            if (type === 'stop_limit') {
+                const stopPrice = this.safeString2 (params, 'stopPx', 'stopPrice');
+                if (stopPrice === undefined) {
+                    throw new ArgumentsRequired (this.id + ' createOrder requires a stop-price argument for ' + type + ' orders');
+                }
+                request['stopPrice'] = stopPrice;
```

Comment:
If they require a `stopPrice` parameter, we don't need a `stopPx` parameter at all. If they require a `stopPx` parameter, then we support both `stopPx` and `stopPrice`.

To clarify: in most cases we will support the exchange-specific parameter name and the CCXT unified parameter name for stopPrice:

```JavaScript
//                                           ↓ exchnage-specific
//                                           ↓
//                                           ↓          ↓ unified
//                                           ↓          ↓
const stopPrice = this.safeValue2 (params, 'stopPx', 'stopPrice');
if (stopPrice !== undefined) {
    params = this.omit (params, [ 'stopPx', 'stopPrice' ]); // omit both params
    request['stopPx'] = stopPrice; // set the exchange-specific parameter name to stopPrice
}
```

In this case, however, the exchange-specific parameter name for the stop price is the same as the unified parameter name, so this exchange does not support `stopPx` at all:

```JavaScript
const stopPrice = this.safeValue (params, 'stopPrice');
if (stopPrice !== undefined) {
    // not necessary, cause it will be done in this.extend upon sending out the request
    // request['stopPrice'] = stopPrice; 
    request['type'] = 'stop_limit';
}
```